app/posts/publish-teacher-training-courses/2018-06-06-school-direct-view.md

```diff
@@ -9,7 +9,7 @@ School Direct routes are the most complex. List schools, show multiple UCAS cour
 We tested this with West London Teaching School Alliance:
 [research session](https://lookback.io/watch/mbc9BNqBJjoRkinAE). We also demoed it to groups of School Direct and SCITT providers in Brighton and Reading. Some provider user needs have been derived from these sessions: [provider user needs](https://docs.google.com/document/d/1Jb5uDZBnSFGcCdl3gAF9ggAo1klsEWL-KeSIGGLJpBE/edit)
 
-An iteration of this was tested with [Kingston School Direct](/publish-teacher-training-courses/kingston-school-direct)
```

Comment:
This one still needs to be ported.

Reply:
Link text restored. Will create an issue for porting this page over.